plugins/server/org.mongodb/mongodb/2.2/manifest.ktor.yaml

```diff
@@ -0,0 +1,13 @@
+name: MongoDB
+description: Adds MongoDB database to your application
+vcsLink: https://github.com/mongodb/mongo
+license: Apache 2.0
+category: Databases
+prerequisites:
+  - routing
+  - kotlinx-serialization
+documentation: documentation.md
+installation:
+  outside_app: Databases.kt
+  source_file_kt: CarsSchema.kt
```

Comment:
It would appear the resolution is case sensitive:

```
java.lang.IllegalArgumentException: Missing install snippet Databases.kt
```

I'd rename the `carsSchema.kt` file to `CarsSchema.kt` since it'll be included separately.  The `databases.kt` one doesn't really matter so long as it matches the reference here.